_locales/it/messages.json

```
@@ -0,0 +1,300 @@
+{
+	"additionalPermissions": {
+		"message": "Permessi aggiuntivi"
+	},
+
+	"all": {
+		"message": "Globale"
+	},
+
+	"alwaysUsesHttp": {
+		"message": "Usa sempre http"
+	},
+
+	"alwaysUsesHttps": {
+		"message": "Usa sempre https"
+	},
+
+	"ask": {
+		"message": "Chiedi"
+	},
+
+	"bookmarks": {
+		"message": "leggere e modificare i segnalibri"
+	},
+
+	"default": {
+		"message": "Predefinito"
+	},
+
+	"enabled": {
+		"message": "Abilitata"
+	},
+
+	"errorHTMLTitle": {
+		"message": "Errore",
+		"description": "The title of error.html."
+	},
+
+	"extensionDescription": {
+		"message": "Apri un link nella barra laterale.",
+		"description": "Description of the extension."
+	},
+
+	"extensionName": {
+		"message": "Apri nella barra laterale",
+		"description": "Name of the extension."
+	},
+
+	"host": {
+		"message": "accedere ai dati dei siti web"
+	},
+
+	"iconImageAlt": {
+		"message": "L'icona di questa estensione.",
+		"description": "The alternate description of the icon of this extention."
+	},
+
+	"indexHTMLCaution": {
+		"message": "Nota: selezionando un url con protocollo https, sarà privilegiata l'opzione https anche se si sceglie http.",
+		"description": "Cautions of this extention."
+	},
+
+	"indexHTMLHeading1": {
+		"message": "1. Cliccare con il tasto destro del mouse su un link o la selezione di un url in formato testo e selezionare \"Apri nella barra laterale\" nel menu contestuale.",
+		"description": "The first heading of index.html."
+	},
+
+	"indexHTMLHeading2": {
+		"message": "2. Scegliere e cliccare sulla voce che si desidera aprire (usare https in caso di dubbi).",
+		"description": "The second heading of index.html."
+	},
+
+	"indexHTMLOthers": {
+		"message": "Ad esempio si possono aprire nella barra laterale un audio, un'immagine, un video, un segnalibro o una scheda.",
+		"description": "others of index.html"
+	},
+
+	"indexHTMLTitle": {
+		"message": "Guida",
+		"description": "The title of index.html."
+	},
+
+	"indexHTMLTutorialVideo": {
+		"message": "https://drive.google.com/file/d/14tn0lZpNIybOt94NmpdqJvWPlKlnLjCm/view",
+		"description": "The url of tutorial video."
+	},
+
+	"initialLocation": {
+		"message": "Pagina iniziale"
+	},
+
+	"initialLocationDescription": {
+		"message": "Premere 'Invio' per salvare. Se lasciato vuoto si resetta allo stato predefinito."
+	},
+
+	"openFromBookmark": {
+		"message": "Apri da un segnalibro"
+	},
+
+	"openFromLink": {
+		"message": "Apri da un link"
+	},
+
+	"openFromPage": {
+		"message": "Apri da una pagina"
+	},
+
+	"openFromSelection": {
+		"message": "Apri da una selezione"
+	},
+
+	"openFromSidebar": {
+		"message": "Apri dalla barra laterale "
```

Please remove unnecessary space.
```suggestion
		"message": "Apri dalla barra laterale"
```